ClusterInfoTest.all_nodes_should_have_the_same_cluster_start_time_and_id_after_master_shutdown_and_new_node_join

```
java.lang.AssertionError: the size of the collection is not correct: found-content:[Member [127.0.0.1]:9806, Member [127.0.0.1]:9808 this, Member [127.0.0.1]:9810] expected:<2> but was:<3>
	at org.junit.Assert.fail(Assert.java:88)
	at org.junit.Assert.failNotEquals(Assert.java:834)
	at org.junit.Assert.assertEquals(Assert.java:645)
	at com.hazelcast.test.HazelcastTestSupport$6.run(HazelcastTestSupport.java:643)
	at com.hazelcast.test.HazelcastTestSupport.assertTrueEventually(HazelcastTestSupport.java:825)
	at com.hazelcast.test.HazelcastTestSupport.assertSizeEventually(HazelcastTestSupport.java:640)
	at com.hazelcast.test.HazelcastTestSupport.assertSizeEventually(HazelcastTestSupport.java:636)
	at com.hazelcast.cluster.ClusterInfoTest.all_nodes_should_have_the_same_cluster_start_time_and_id_after_master_shutdown_and_new_node_join(ClusterInfoTest.java:102)
```

https://hazelcast-l337.ci.cloudbees.com/job/Hazelcast-3.x-OpenJDK8-Quality-Outreach/com.hazelcast$hazelcast/234/testReport/junit/com.hazelcast.cluster/ClusterInfoTest/all_nodes_should_have_the_same_cluster_start_time_and_id_after_master_shutdown_and_new_node_join/

```
java.lang.AssertionError: the size of the collection is not correct: found-content:[Member [127.0.0.1]:8248, Member [127.0.0.1]:8249 this, Member [127.0.0.1]:8250] expected:<2> but was:<3>
	at org.junit.Assert.fail(Assert.java:88)
	at org.junit.Assert.failNotEquals(Assert.java:834)
	at org.junit.Assert.assertEquals(Assert.java:645)
	at com.hazelcast.test.HazelcastTestSupport$6.run(HazelcastTestSupport.java:643)
	at com.hazelcast.test.HazelcastTestSupport.assertTrueEventually(HazelcastTestSupport.java:825)
	at com.hazelcast.test.HazelcastTestSupport.assertSizeEventually(HazelcastTestSupport.java:640)
	at com.hazelcast.test.HazelcastTestSupport.assertSizeEventually(HazelcastTestSupport.java:636)
	at com.hazelcast.cluster.ClusterInfoTest.all_nodes_should_have_the_same_cluster_start_time_and_id_after_master_shutdown_and_new_node_join(ClusterInfoTest.java:102)
```

https://hazelcast-l337.ci.cloudbees.com/view/Hazelcast/job/Hazelcast-3.x-ZuluJDK8/lastCompletedBuild/com.hazelcast$hazelcast/testReport/com.hazelcast.cluster/ClusterInfoTest/all_nodes_should_have_the_same_cluster_start_time_and_id_after_master_shutdown_and_new_node_join/

--- a/hazelcast/src/test/java/com/hazelcast/cluster/ClusterInfoTest.java
+++ b/hazelcast/src/test/java/com/hazelcast/cluster/ClusterInfoTest.java
@@ -52,9 +52,9 @@ public class ClusterInfoTest extends HazelcastTestSupport {
         HazelcastInstance h2 = factory.newHazelcastInstance();
         HazelcastInstance h3 = factory.newHazelcastInstance();
 
-        assertSizeEventually(3, h1.getCluster().getMembers());
-        assertSizeEventually(3, h2.getCluster().getMembers());
-        assertSizeEventually(3, h3.getCluster().getMembers());
+        assertClusterSizeEventually(3, h1);
+        assertClusterSizeEventually(3, h2);
+        assertClusterSizeEventually(3, h3);
 
         Node node1 = TestUtil.getNode(h1);
         Node node2 = TestUtil.getNode(h2);
@@ -86,9 +86,9 @@ public class ClusterInfoTest extends HazelcastTestSupport {
         HazelcastInstance h2 = factory.newHazelcastInstance();
         HazelcastInstance h3 = factory.newHazelcastInstance();
 
-        assertSizeEventually(3, h1.getCluster().getMembers());
-        assertSizeEventually(3, h2.getCluster().getMembers());
-        assertSizeEventually(3, h3.getCluster().getMembers());
+        assertClusterSizeEventually(3, h1);
+        assertClusterSizeEventually(3, h2);
+        assertClusterSizeEventually(3, h3);
 
         Node node1 = TestUtil.getNode(h1);
         final ClusterServiceImpl clusterService = node1.getClusterService();
@@ -99,7 +99,7 @@ public class ClusterInfoTest extends HazelcastTestSupport {
         assertTrue(clusterUpTime > 0);
         assertTrue(node1.isMaster());
         h1.shutdown();
-        assertSizeEventually(2, h2.getCluster().getMembers());
+        assertClusterSizeEventually(2, h2);
 
         HazelcastInstance h4 = factory.newHazelcastInstance();
 

